content/en/docs/setup/install/multicluster/_index.md

```diff
@@ -9,13 +9,715 @@ aliases:
 keywords: [kubernetes,multicluster]
 test: n/a
 ---
+When installing Istio across multiple clusters, there are more things to
+consider than with a [single cluster installation](/docs/setup/getting-started).
+
+There is no default installation procedure for a multicluster mesh, as it will
+be different based on your cluster and network architecture. We therefore
+provide options for:
```

Comment:
```suggestion
Although there is no default installation procedure for a multicluster mesh, as it will
be different based on your cluster and network architecture, this guide covers some
of the most common setups:
provide options for:
```